internal/command/deploy/machines.go

```diff
@@ -317,49 +315,11 @@ func (md *machineDeployment) setMachinesForDeployment(ctx context.Context) error
 
 	// migrate non-platform machines into fly platform
 	if len(machines) == 0 {
-		terminal.Debug("Found no machines that are part of Fly Apps Platform. Check for other machines...")
+		terminal.Debug("Found no machines that are part of Fly Apps Platform. Checking for active machines...")
 		machines, err = md.flapsClient.ListActive(ctx)
 		if err != nil {
 			return err
 		}
-		if len(machines) > 0 {
-			rows := make([][]string, 0)
-			for _, machine := range machines {
-				var volName string
-				if machine.Config != nil && len(machine.Config.Mounts) > 0 {
-					volName = machine.Config.Mounts[0].Volume
-				}
-
-				rows = append(rows, []string{
-					machine.ID,
-					machine.Name,
-					machine.State,
-					machine.Region,
-					machine.ImageRefWithVersion(),
-					machine.PrivateIP,
-					volName,
-					machine.CreatedAt,
-					machine.UpdatedAt,
-				})
-			}
-			terminal.Warnf("Found %d machines that are not part of the Fly Apps Platform:\n", len(machines))
-			_ = render.Table(iostreams.FromContext(ctx).Out, fmt.Sprintf("%s machines", md.app.Name), rows, "ID", "Name", "State", "Region", "Image", "IP Address", "Volume", "Created", "Last Updated")
-			if !md.autoConfirmAppsV2Migration {
```

Comment:
Should we remove the `autoConfirmAppsV2Migration` field from the machine deployment struct?

Reply:
nice catch, thx! will remove